modules/sagemaker/sagemaker-templates-service-catalog/settings.py

```diff
@@ -50,6 +50,11 @@ class ModuleSettings(CdkBaseSettings):
     sagemaker_domain_id: str = Field(default="")
     sagemaker_domain_arn: str = Field(default="")
 
+    repository_type: str = Field(default="GitHub")
```

Comment:
Should this be an enum to have only a limited number of values?

Reply:
Added enum for types of repositories.